Bottom: a590756ec8b824c6749c51d3f373ca6a82f872fc
Top:    fe3eb2f03afc28b20550e9d83377c534a3f03f33
Author: student <student@ReliableEmbeddedSystems.com>
Date:   2019-09-10 15:22:51 +0200

added kubernetes user/passwd, probably for different layer

Signed-off-by: student <student@ReliableEmbeddedSystems.com>


---

diff --git a/recipes-containers/kubernetes/kubernetes_%.bbappend b/recipes-containers/kubernetes/kubernetes_%.bbappend
new file mode 100644
index 0000000..ad2a57e
--- /dev/null
+++ b/recipes-containers/kubernetes/kubernetes_%.bbappend
@@ -0,0 +1,10 @@
+inherit useradd
+
+# created with 
+# mkpasswd kubernetes
+# Sq1x2H9sA4eWM
+PASSWORD ?= "Sq1x2H9sA4eWM"
+USERADD_PACKAGES = "${PN}"
+USERADD_PARAM_${PN} = "--system --create-home \
+                       --password ${PASSWORD} \
+                       --user-group ${PN}"
